dev/archery/archery/bot.py

```diff
@@ -229,7 +229,7 @@ def submit(obj, tasks, groups, params, arrow_version):
     """
     Submit crossbow testing tasks.
 
-    See groups defined in arrow/dev/tasks/tests.yml
```

Comment:
`tests.yml` doesn't exist, looks it should be `tasks.yml`
https://github.com/apache/arrow/tree/master/dev/tasks